src/datagen/generated/domum_ornamentum/assets/domum_ornamentum/models/block/corner_panel/corner_panel_outer.json

```diff
@@ -0,0 +1,4 @@
+{
+  "parent": "domum_ornamentum:block/corner_panel/corner_panel_outer_spec",
```

Comment:
I'm already skeptical about corner panels in general. But I don't see why we need two variants. The outer corner doesn't seem useful to me. Having an additional entry for this?

Reply:
Otherwise it would just be a single new item then, right?

Reply:
I think because the shape is different it makes more sense having them separated, otherwise how do I render the item icons?

Reply:
I mean, just not have it at all. Its building utility is minimal.

Reply:
I made it so you could work away the few rough edges in corners that currently aren't possible, when 2 panels meet eachother on the outer corners there's that small indent, that's where it's supposed to go

Reply:
The shape type doesn't even add that much extra complexity on top of what there already is right now, and it's still possible to have more corner types to fill gaps in.
If the sole issue is the shape type is adding more complexity, it isn't going to change much if I remove it.
Not to mention I'll have to upset all of the happy Patrons again who have seen the spoiler and are happy about getting it.

Reply:
Maybe we shouldn't post things there that lead already voiced concerns about previously.

Reply:
and I doubt most patreons care about the slight filler variant.

Reply:
There are 2 items added to the Cutter here right. The inner and the outer variant?

Reply:
Yes

Reply:
So, by removing this corner variant, that is much less useful, we add only a single variant, which makes this overall much more acceptable as a tradeoff.